modules/shortcodes.php

```diff
@@ -194,3 +194,37 @@ function wpcom_shortcodereverse_parseattr( $attrs ) {
 }
 
 jetpack_load_shortcodes();
+
+/**
+ * Load Gutenberg blocks
+ *
+ * @since 5.4
+ */
+function gutenpack_editor_assets_fold() {
+	wp_enqueue_style(
+		'gutenpack-editor',
+		plugins_url( 'modules/shortcodes/css/gutenpack.css', JETPACK__PLUGIN_FILE ),
```

Comment:
Should we flag this as RTL friendly so Janus's generated rtl stylesheet can replace it?